bundles/tampere/content-editor/view/SideContentEditor.js

```diff
@@ -600,24 +816,29 @@ Oskari.clazz.define('Oskari.tampere.bundle.content-editor.view.SideContentEditor
          */
         prepareRequest: function (geometries, deleteFeature) {
             var me = this;
-            var requestData = {};
-            requestData.featureFields = [];
+            var requestData = [];
+
+            var feature = {};
+            feature.featureFields = [];
             var featureData = me._getFeatureData();
             featureData.splice(0, 1);
-            requestData.featureFields = featureData;
-
-            requestData.featureId = (me._getFeatureData().length > 0 ? me._getFeatureData()[0].value : null);
-            requestData.layerId = me.selectedLayerId;
-            requestData.srsName = this.sandbox.getMap().getSrsName();
-            requestData.geometries = {};
-            requestData.geometries.data = [];
-            if (me.operationMode == 'edit' || deleteFeature == true) {
-                me._fillLayerGeometries(requestData.geometries);
-            } else if (geometries != null) {
-                me._cloneGeometries(requestData, geometries);
+            feature.featureFields = featureData;
+
+            feature.featureId = (me._getFeatureData().length > 0 ? me._getFeatureData()[0].value : null);
+            feature.layerId = me.selectedLayerId;
+            feature.srsName = this.sandbox.getMap().getSrsName();
+            feature.geometries = {};
+            feature.geometries.data = [];
+            if (!me.editMultipleFeatures && (me.operationMode == 'edit' || me.operationMode == 'create' || deleteFeature == true)) {
+                me._fillLayerGeometries(feature.geometries);
+            } else if(me.editMultipleFeatures) {
+                // not allowed change geometries when multiple edit
+                delete feature.geometries;
```

Comment:
This looks like the place that the payload should be generated for multiEdit. Now the old code is using this as prepare and the multiEdit does some additional preparing on the sendRequest()